### ISSUE TITLE
fix some links

### ISSUE THREAD

**Issue by MarcAntoine-Arnaud** [rudof-project/rudof#136]
on crates.io some links are broken: https://crates.io/crates/rudof

**Comment by labra** [comment]
Indeed, something I am currently not sure about is what is the `rudof` crate at all because the latest version has been published 2 months ago but we are currently not publishing a single `rudof` crate but more fine-grained crates like [rudof-cli](https://crates.io/crates/rudof_cli) or [shacl_ast](https://crates.io/crates/shacl_ast) which are more up-to-date.

Which are the links that are broken?

**Comment by MarcAntoine-Arnaud** [comment]
Ho yeah, I have opened this issue just after you changed the name.
So it's mostly here: [https://crates.io/crates/rudof](https://crates.io/crates/rudof)

But as it's a not used crate, I suggest:
- remove published crates
- publish a crate that just redirect to other crate and do right documentation/repository

**Comment by labra** [comment]
I think that having an façade `rudof` crate with a functionality similar to that of the `rudof_cli` but intended to be used programmatically would make sense 

In that way, we could encourage the Rust users to use the methods from that crate trying to define a more stable API on that module. 

I am hesitating about the name of the crate, we could call it `rudof`, but maybe I will call it `rudof_lib` by now, reserving `rudof`, and mainly because in that way, the structure of the project I think is cleaner, as we will have a folder for each crate.

**Comment by MarcAntoine-Arnaud** [comment]
For me it can be `rudof` the main crate.
A crate can be a lib and may includes also a `bin` folder with some executables (as mentioned here https://doc.rust-lang.org/cargo/reference/cargo-targets.html#binaries).

It means you have only one crate that reference the cli and the lib.

**Comment by labra** [comment]
Yes, indeed, I think we could do it. I tried to keep `rudof_cli` and `rudof_lib` separated to avoid conflicts, but maybe we should consider grouping them back together. 

I will do it once I am sure how to do it properly...
